lib/commands/misc/rmrole.js

```diff
@@ -0,0 +1,61 @@
+const { Command } = require('discord.js-commando');
+const { GENERAL_ID } = require('../../constants.js');
+const { getChannel, inCommandChannel } = require('./../../util.js');
+
+module.exports = class RemoveRoleCommand extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'rmrole',
+			group: 'misc',
+			memberName: 'rmrole',
+			description: 'Remove a self-assigned role from yourself.',
+			details: '',
+			args: [
+				{
+					key: 'role',
+					prompt: 'Remove a role!',
+					type: 'string',
+					infinite: false,
+					wait: 1
+				}
+			]
+		});
+		this.queues = {};
+	};
+
+  async run(msg, args) {
+
+		const roleArg = args.role;
+		const memberRoles = msg.member.roles;
+		const channel = this.client.channels.get(getChannel(BOT_STUFF_ID));
+
+		if (!inCommandChannel(msg)) {
+			return msg.reply("Please use this command in the designated bot commands channel!");
+		}
+
+		if (roleArg.length < 1) {
+			return msg.reply("Make sure to enter a role to remove! Ex: r!rmrole Red");
+		}
+
+    const role = msg.guild.roles.find(r => r.name == roleArg);
+
+    // Prevent removal of required roles (Can be updated in the future if more required roles are ever added).
+    if (roleArg.toLowerCase() === "member") {
+      msg.reply("You can't remove that role from yourself!");
+      return;
```

Comment:
code should be consistent! `return msg.reply("You can't remove that role from yourself!");`